theme/keyframes.ts

```diff
@@ -8,6 +8,14 @@ export const keyframes: CssKeyframes = {
     from: { opacity: 1, transform: 'translateY(0)' },
     to: { opacity: 0, transform: 'translateY(4px)' },
   },
+  slideDown: {
```

Comment:
We do have a `slideUpAndFade` and `slideDownAndFade` being used in other UI components (dropdown, select, and tooltip) ...wondering about naming here bc specific to this feature with the `maxHeight`?

Reply:
hm, how do you think better to name? 🤔

Reply:
Not necessary to change, but thought maybe just adding the `maxHeight` to the ones we already have might work?  Not critical to address.